templates/cart.liquid

```diff
@@ -1,57 +1,64 @@
 <section id="cart" class="font-uppercase">
-  <h1 class="text-align-center padding-top-small padding-bottom-small">Cart</h1>
   <div class="container">
+    <div class="border-bottom padding-top-small padding-bottom-small">
+      <h1 class="font-size-mid text-align-center">Cart</h1>
+    </div>
     {% if cart.item_count > 0 %}
-    <form action="/cart" method="post" id="cart" class="cart grid-row">
-      <div class="grid-item item-s-12 item-l-6 no-gutter grid-row align-items-start">
-        {% for item in cart.items %}
-        {% assign variant_price = item.price | money %}
-        <div id="cart-item-{{item.variant.id}}" class="cart-item grid-item item-s-12 padding-top-small padding-bottom-small no-gutter grid-row align-items-center">
-          <div class="grid-item item-s-3 item-l-2">
-            <input type="text" size="4" pattern="[0-9]*" name="updates[{{item.variant.id}}]" id="quantity-{{ item.variant.id }}" class="form-element quantity" value="{{ item.quantity }}" min="1" onfocus="this.select();"/>
-          </div>
-          <div class="grid-item item-s-2 item-l-3">
-            <a href="{{ item.product.url }}">
-              {% if item.variant.image != blank %}
-                <img src="{{ item.variant.image | img_url: '300x300' }}" alt="{{ item.title | escape }}" />
-              {% else %}
-                <img src="{{ item.image | img_url: '300x300' }}" alt="{{ item.title | escape }}" />
-              {% endif %}
-            </a>
-          </div>
-          <div class="grid-item item-s-4">
-            <div class="margin-bottom-micro">
-              <a href="{{ item.product.url }}">{{ item.product.title }}</a>
+    <form action="/cart" method="post" id="cart" class="cart full-height-after-title">
+      <div id="cart-items-column" class="grid-item item-s-12 item-l-6 flex-grow no-gutter grid-column border-left-half justify-between">
+        <div class="grid-item no-gutter flex-row font-size-small">
+          {% for item in cart.items %}
+          {% assign variant_price = item.price | money %}
+          <div id="cart-item-{{item.variant.id}}" class="cart-item grid-item item-s-12 padding-top-small padding-bottom-small no-gutter grid-row align-items-center border-bottom">
+            <div class="grid-item item-s-2 item-l-2 text-align-center">
+              <input type="text" size="4" pattern="[0-9]*" name="updates[{{item.variant.id}}]" id="quantity-{{ item.variant.id }}" class="form-element quantity" value="{{ item.quantity }}" min="1" onfocus="this.select();"/>
```

Comment:
should this limit to 2 for likelyhood and also style if accidentally filled?

Reply:
yea limit to 2 is good.

style if accidentally filled? clarify?

Reply:
just if the user held the button down or something and entered 4 digits it looked bad.